boards/arm/lpcxpresso55S69/doc/index.rst

```diff
@@ -0,0 +1,210 @@
+- Onboard, high-speed USB, Link2 debug probe with CMSIS-DAP and SEGGER J-Link
+  protocol options
+- UART and SPI port bridging from LPC55S69 target to USB via the onboard debug
+  probe
+- Hardware support for external debug probe
+- 3 x user LEDs, plus Reset, ISP (3) and user buttons
+- Micro SD card slot (4-bit SDIO)
+- NXP MMA8652FCR1 accelerometer
+- Stereo audio codec with line in/out
+- High and full speed USB ports with micro A/B connector for host or device
+  functionality
+- MikroEletronika Click expansion option
+- LPCXpresso-V3 expansion option compatible with Arduino UNO
+- PMod compatible expansion / host connector
+
+For more information about the LPC55S69 SoC and LPCXPRESSO55S69 board:
+
+- `LPC55S69 SoC Website`_
+- `LPC55S69 Datasheet`_
+- `LPC55S69 Reference Manual`_
+- `LPCXPRESSO55S69 Website`_
+- `LPCXPRESSO55S69 User Guide`_
+- `LPCXPRESSO55S69 Schematics`_
+
+Supported Features
+==================
+
+The lpcxpresso55s69 board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| IOCON     | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| USART     | on-chip    | serial port-polling                 |
++-----------+------------+-------------------------------------+
+
+The default configuration only enables the first core for now and can
+be found in the following defconfig file:
+
+	``boards/arm/lpcxpresso55s69/lpcxpresso55s69_cpu0_defconfig``
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+The LPC55S69 SoC has IOCON registers, which can be used to configure the
+functionality of a pin.
+
++---------+-----------------+----------------------------+
+| Name    | Function        | Usage                      |
++=========+=================+============================+
+| PIO0_29 | USART           | USART RX                   |
++---------+-----------------+----------------------------+
+| PIO0_30 | USART           | USART TX                   |
++---------+-----------------+----------------------------+
+| PIO1_4 _| GPIO            | RED LED                    |
++---------+-----------------+----------------------------+
+| PIO1_6  | GPIO            | BLUE_LED                   |
++---------+-----------------+----------------------------+
+| PIO1_7_ | GPIO            | GREEN LED                  |
```

Comment:
If you need to have an underscore here, write this as ``PIO1_7\_`` (same on line 88 above).  Otherwise delete both of these trailing underscores.